### DISCUSSION TITLE
Mark/pin Most used TOTP entries on the top

### DISCUSSION THREAD

**Discussion by dnet890** [ente-io/ente#1307] [Enhancements]
Hi
I found Ente app is quite good. But, is it possible to pin certain TOTP entry on the top?

**Comment by nktnet1** [comment]
Having a custom sort order as suggested in #1614 would be great.

One workaround is to rename the service with a number prefix, e.g. "1. Some Service", "2. Some other Service", as they will be alphabetically sorted.

**Reply by nktnet1** [reply]
Update - custom sort has been made available. See pull request [#4072](https://github.com/ente-io/ente/pull/4072), released in [v4.1.5](https://github.com/ente-io/ente/releases/tag/auth-v4.1.5).

![instructions](https://github.com/user-attachments/assets/52e6e862-1a9c-412f-9f6a-a4b62dc23577)

**Comment by ua741** [comment]
In the latest version, you can now add tags and pin specific entries.
